wire-runtime/src/main/java/com/squareup/wire/ProtoReader.java

```diff
@@ -122,10 +122,24 @@ public void endLengthDelimited(long token) throws IOException {
   }
 
   /**
-   * Read the tag of the next field. Use {@link #peekType()} after calling this method to query its
-   * type.
+   * Reads and returns the next tag of the message, or -1 if there are no further tags. This
+   * silently skips tags whose types are groups. Use {@link #peekType()} after calling this method
+   * to query its type.
```

Comment:
Switch the 2nd and 3rd sentences. Otherwise the "its" in the last sentence is ambiguous.